dnf-behave-tests/features/fail-safe-cache.feature

```diff
@@ -0,0 +1,308 @@
+Feature: Tests for the correct creation and usage of the modulefailsafe file
+
+
+Background: Copy the dnf-ci-fedora modular repository (to allow modulemd removal and so on) and enable nodejs:5 (containing the oldest nodejs package) -> this makes fail-safe copy of nodejs:5
+  Given I copy directory "{context.dnf.repos_location}/dnf-ci-fedora-modular" to "/temp-repos/dnf-ci-fedora-modular"
+   And I create and substitute file "/etc/yum.repos.d/dnf-ci-fedora-modular.repo" with
+       """
+       [dnf-ci-fedora-modular]
+       name=dnf-ci-fedora-modular
+       baseurl={context.dnf.installroot}/temp-repos/dnf-ci-fedora-modular
+       enabled=1
+       gpgcheck=0
+       skip_if_unavailable=1
+       """
+    And I copy file "{context.dnf.reposdir}/dnf-ci-fedora.repo" to "/etc/yum.repos.d/dnf-ci-fedora.repo"
+    And I copy file "{context.dnf.reposdir}/dnf-ci-fedora-modular-updates.repo" to "/etc/yum.repos.d/dnf-ci-fedora-modular-updates.repo"
+    And I copy file "{context.dnf.reposdir}/dnf-ci-fedora-updates.repo" to "/etc/yum.repos.d/dnf-ci-fedora-updates.repo"
+    And I do not set reposdir
+    And I use the repository "dnf-ci-fedora-modular"
+    And I use the repository "dnf-ci-fedora-modular-updates"
+    And I use the repository "dnf-ci-fedora"
+
+
+# nodejs:
+#     dnf-ci-fedora:
+#         nodejs-1:5.12.1-1.fc29.x86_64
+#     dnf-ci-fedora-modular:
+#         nodejs-1:5.3.1-1.module_2011+41787af0.x86_64
+#         nodejs-1:8.11.4-1.module_2030+42747d40.x86_64
+#         nodejs-1:10.11.0-1.module_2200+adbac02b.x86_64
+#         nodejs-1:11.0.0-1.module_2311+8d497411.x86_64
+#         - nodejs:5 - nodejs-1:5.3.1-1.module_2011+41787af0.x86_64
+#         - nodejs:8 - nodejs-1:8.11.4-1.module_2030+42747d40.x86_64
+#         - nodejs:10 - nodejs-1:10.11.0-1.module_2200+adbac02b.x86_64
+#         - nodejs:11 - nodejs-1:11.0.0-1.module_2311+8d497411.x86_64
+#     dnf-ci-fedora-modular-updates:
+#         nodejs-1:10.14.1-1.module_2533+7361f245.x86_64
+#         nodejs-1:11.1.0-1.module_2379+8d497405.x86_64
+#         nodejs-1:12.1.0-1.module_2379+8d497405.x86_64
+#         nodejs-1:8.11.4-1.module_2030+42747d40.x86_64
+#         nodejs-1:8.14.0-1.module_2030+42747d40.x86_64
+#         - nodejs:8 - nodejs-1:8.11.4-1.module_2030+42747d40.x86_64
+#         - nodejs:10 - nodejs-1:10.14.1-1.module_2533+7361f245.x86_64
+#         - nodejs:11 - nodejs-1:11.1.0-1.module_2379+8d497405.x86_64
+#         - nodejs:12 - nodejs-1:12.1.0-1.module_2379+8d497405.x86_64
```

Comment:
Please what the purpose of comment? Please add a description as the first comment line